src/main/java/net/snowflake/client/jdbc/SnowflakeDatabaseMetaData.java

```diff
@@ -1418,7 +1418,7 @@ public ResultSet getTables(
     final Pattern compiledSchemaPattern = Wildcard.toRegexPattern(schemaPattern, true);
     final Pattern compiledTablePattern = Wildcard.toRegexPattern(tableNamePattern, true);
 
-    showCommand = null;
+    String showCommand = "";
```

Comment:
I think this issue is there for other metadata functions as well, just verify once

Reply:
Why change default starting value from null to empty string?

Reply:
Yeah, it has another one in **Line 1611**, but these changes will break the test of `DatabaseMetaDataInternalLatestIT`

Reply:
Is the test the only reason for the class variable? If so, we can set the class variable once the local variable is done being set.
```
this.showCommand = showCommand
```

Reply:
Yes, it's only for testing usage.